src/Database/Connection.php

```diff
@@ -814,13 +814,35 @@ public function logQueries($enable = null)
     public function logger($instance = null)
     {
         if ($instance === null) {
-            if ($this->_logger === null) {
```

Comment:
@burzum personally, I prefer to have blank line after inheriting, as it makes it more visible that I inherit and what I overwrite/add.